packages-native/actions/src/calendar/ViewCalendarEvent.ts

```diff
@@ -1,28 +0,0 @@
-// This file was generated by Mendix Modeler.
```

Comment:
Why do we get rid of the calendar related actions?

Reply:
Because we don’t have this jsactions exposed in our modules